params/config.go

```diff
@@ -31,21 +31,62 @@ import (
 // Genesis hashes to enforce below configs on.
 var (
 	MainnetGenesisHash      = common.HexToHash("// todo generate new hash for mainnet") // Mainnet genesis hash to enforce below configs on
```

Comment:
```suggestion
	CypressGenesisHash      = common.HexToHash("// todo generate new hash for mainnet") // Mainnet genesis hash to enforce below configs on
```
For alignment with `BaobabGenesisHash`?

Reply:
thanks, reflected at https://github.com/klaytn/klaytn/pull/882/commits/16a16c3814a573dbc65e3c774b753bd33d918755

Reply:
I forced push the commit.. Could you review again?